src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -119,6 +130,31 @@
                 </a>
               </td>
             {% endif %}
+
+            <td class="text-right">
+              <span class="partial-refund{% if product.isRefundable() == false %} hidden{% endif %}">
+                <div class="form-group">
+                  <div class="col-lg-4">
+                    <div class="input-group">
+                      <strong>{{ form_label(partialRefundForm['quantity_' ~ product.orderDetailId]) }}</strong>
+                      {{ form_widget(partialRefundForm['quantity_' ~ product.orderDetailId]) }}
```

Comment:
This is a weird syntax, I think if you use a CollectionType in the form type you can have cleaner way to access each product sub form